plugins/offers_offer.c

```diff
@@ -732,8 +728,7 @@ struct command_result *json_invoicerequest(struct command *cmd,
 
 	/* FIXME: We only set blinded path if private, we should allow
 	 * setting otherwise! */
```

Comment:
```suggestion
	/* FIXME: We only set blinded path if private or when no advertised address is available, we should allow
	 * setting otherwise! */
```

probably good to point this out too?